source/content.css

```diff
@@ -971,3 +971,13 @@ body > .footer li a {
 	background: #9c95b3;
 	cursor: pointer;
 }
+
+blockquote {
+	max-height: 30.5em;
+	overflow-y: auto;
+}
+
+pre {
```

Comment:
These can be combined:

```css
blockquote,
pre {
	max-height: 30.5em;
	overflow-y: auto;
}
```

Reply:
It should also be limited to comments so that other regular files and readmes aren’t affected.